moveit_planners/ompl/ompl_interface/include/moveit/ompl_interface/model_based_planning_context.h

```diff
@@ -242,9 +241,19 @@ class ModelBasedPlanningContext : public planning_interface::PlanningContext
                           const moveit_msgs::Constraints& path_constraints, moveit_msgs::MoveItErrorCodes* error);
   bool setPathConstraints(const moveit_msgs::Constraints& path_constraints, moveit_msgs::MoveItErrorCodes* error);
 
-  void setConstraintsApproximations(const ConstraintsLibraryConstPtr& constraints_library)
+  void setConstraintsApproximations(const ConstraintsLibraryPtr& constraints_library)
   {
-    spec_.constraints_library_ = constraints_library;
+    constraints_library_ = constraints_library;
+  }
+
+  ConstraintsLibraryPtr getConstraintsLibrary()
```

Comment:
```suggestion
  ConstraintsLibraryPtr getConstraintsLibraryNonConst()
```
This is a standard used in a lot of moveit code, what do you think?

Reply:
Good point, I've made the distinction.